example_workflow/environment.yml

```
@@ -0,0 +1,12 @@
+name: pymser
+channels:
+  - conda-forge
+dependencies:
+  - python >= 3.10
+  - pip
+  - numpy
+  - pandas
+  - gemmi
+  - pip:
+    - pymser
+    - RASPA2
```

I am trying to bring this Python environment closer to that of pyMSER itself, which uses `Pipfile` instead of Anaconda and is based on Python 3.9 instead of 3.10, to minimise the chance of incompatibility.

This would be, in principle, the literal translation of the `environment.yml` into a `Pipfile` format. Can you please confirm whether this works for you?

```suggestion
[[source]]
url = "https://pypi.org/simple"
verify_ssl = true
name = "pypi"

[requires]
python_version = "3.9"

[packages]
gemmi = "*"
numpy = "*"
pandas = "*"
pip = "*"
pymser = "*"
raspa2 = "*"
```

---

For some reason it did not work on my computer:

> Installing initially failed dependencies...

It could be just a matter of configure pipenv (I've never used it).